src/cascadia/TerminalSettingsModel/TerminalSettings.cpp

```diff
@@ -110,7 +107,27 @@ namespace winrt::TerminalApp::implementation
             }
         }
 
-        return { profileGuid, settings };
+        return settings;
+    }
+
+    // Method Description:
+    // - Creates a projectedType version of a child of this TerminalSettings.
+    // - We can't just expose CreateChild() in the IDL because it returns the wrong type.
+    Model::TerminalSettings TerminalSettings::MakeChild() const
+    {
+        return *CreateChild();
+    }
+
+    // Method Description:
+    // - Sets our parent to the provided TerminalSettings
+    // Arguments:
+    // - parent: our new parent
+    void TerminalSettings::SetParent(Model::TerminalSettings parent)

```

Comment:
```suggestion
    void TerminalSettings::SetParent(const Model::TerminalSettings& parent)
```

Avoid an extra reference count.